borgmatic/tests/unit/borg/test_create.py

```diff
@@ -344,3 +312,41 @@ def test_create_archive_with_glob_should_call_borg_with_expanded_directories():
         },
         storage_config={},
     )
+
+
+def test_create_archive_with_archive_name():
```

Comment:
Thanks for adding automated tests!

However I don't think this particular test adds a whole lot versus the test immediately below. I suppose it would catch a regression in which we treated literal archive names differently from placeholder archive names, but that's it.

If you do want to keep this test, I would recommend making its name clearly reflect what exactly it's testing for. Example: `test_create_archive_with_archive_name_passes_through_to_borg()`.